.snapcraft/snap/snapcraft.yaml

```
@@ -7,7 +7,7 @@
 # 5.  `snapcraft snap`
 
 name: rocketchat-server
-version: #{RC_VERSION}
+version: 2.1.0 
```

```suggestion
version: #{RC_VERSION}
```
Before we merge we'll need to make sure to change this back. Can leave for now while testing